make_release.sh

```diff
@@ -48,6 +48,9 @@ patch -d work <FontConversion/RemoveGlyphWidthLimit.patch
 patch -d work <ChangeResolution/ChangeResolution.patch
 patch -d work <LanguageServerProtocolHelper/LSPHelper.patch
 patch -d work <KernelDebugger/RS232.patch
+patch -d work <EditImprovements/Edit.1r.patch
+cp -p UTF8CharsetLite/EditU.Mod.txt work/
+patch -d work <EditImprovements/EditU.0.patch
```

Comment:
The `cp` should go after line 14 and needs to be excluded in line 78. Or (better) the `patch` should go after line 81.